pkg/graph/resolvers/resolver_test.go

```diff
@@ -28,11 +30,32 @@ func (suite *ResolverSuite) SetupTest() {
 	err := suite.testConfigs.Store.TruncateAllTablesDANGEROUS(suite.testConfigs.Logger)
 
 	//GET USER ACCOUNT EACH TIME!
-	princ := getTestPrincipal(suite.testConfigs.Store, suite.testConfigs.UserInfo.Username)
+	princ := suite.GetTestPrincipal(suite.testConfigs.Store, suite.testConfigs.UserInfo.Username)
 	suite.testConfigs.Principal = princ
 	assert.NoError(suite.T(), err)
 }
 
+func (suite *ResolverSuite) GetTestPrincipal(store *storage.Store, userName string) *authentication.ApplicationPrincipal {
```

Comment:
```suggestion
func (suite *ResolverSuite) getTestPrincipal(store *storage.Store, userName string) *authentication.ApplicationPrincipal {
```

Thinking it might make sense to not have this method exported -- does anything outside the resolver suite need it?

Reply:
You are right, nothing should use it outside the suite, so that makes sense. I'll make the change

 Interestingly, if you reference the suite in the same package, but not in a test file, you get a GO-Static check failure.